docs/appendices/release-notes/5.4.4.rst

```diff
@@ -90,3 +90,6 @@ Fixes
 	       END
     FROM (select 'value1' as col1) a;
 
+- Fixed an issue that caused :ref:`column policy <type-object-column-policy>`
+   of object types created by ``CREATE TABLE AS`` to be overridden to
```

Comment:
here and below, please fix indentation.

Reply:
Can you please also fix the identation (tabs are used instead of whitespaces in the CASE example for the changelog entry above? (I've already seen that for the corresponding 5.4 backport PR and fixed it there, but not yet for master).
Thx!

Reply:
Thanks done, also I do not see an update on `5.4` I will just backport this as well.